RPIPs/RPIP-53.md

```diff
@@ -183,5 +183,136 @@ v10 has been implemented into a Smart Node branch [here](https://github.com/rock
 Aside from variable names, the actual functional changes are very small. It just checks to see if a node has an RPL withdrawal address set, and if so, it directs the RPL earnings to the RPL withdrawal address and the ETH earnings to the primary withdrawal address instead.
 
 
+## Appendix A: Setting the RPL Withdrawal Address to a Registered Node
+
+The results of setting your RPL withdrawal address to a registered node address can be somewhat confusing, so the scenarios are spelled out explicitly here to dispel any uncertainty.
+
+
+### Scenario 1
+
+In this scenario, we have the following setup:
+
+```
+Node A: primary A, RPL B
+Node B: primary C, RPL not set
+```
+
+After a rewards event is triggered, the following claimers will be produced in the tree:
+
+```
+Entry A (A's ETH, A's RPL)
+
+Entry B (B's ETH, B's RPL)
+```
+
+- Eligible claimers for Entry A and Entry B are both determined at the time of claiming.
+
+Note that if this is the scenario at the time of claiming Entry A, then only B can claim Entry A. Even though B has a primary withdrawal address set (to C), B will still receive the rewards upon claiming. The contract doesn't look at B's withdrawal addresses when it's claiming for A as A's RPL withdrawal address, because the contracts don't check whether or not B is a node during that process.
+This is the same with or without v10, but is included here for posterity. 
+
```

Comment:
"Entry B is straightforward following Houston rules: if the scenario is as described at the time of claiming, Entry B can be claimed by B or C and all funds will go to C."

May be nice to add this just to be explicit about what the normal Houston idea is (referenced several times below too).

Reply:
Ok, updated with a slight variant of that to clarify how both entries work under Houston rules.